### PR TITLE
1.3.0

### PR DESCRIPTION
**Pull request by jalakoo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,28 +7,219 @@
...
+* Collect dispatch service
+* Tag Management dispatch service
+* Lifecycle autotracking
+* Limited UI autotracking
+* Custom Persistence
+* Tealium Data
+
+
+## DATA LAYER DEPRECATIONS
+The following variables/ event-attributes are marked for deprecation in a future release:
```
Comment (tamayok) [inline README.md]: Change to a Note. Also actually remove the variables

Comment (tamayok) [inline README.md]: Add deprecate message

File: README.md [hunk 2]
```diff
@@ -7,28 +7,219 @@
...
+
+Watch this video for an overview of the installation and setup of the code:
+
+[![Swift Implementation Video](http://res.cloudinary.com/dfpz40r7j/image/upload/v1498577063/SwiftImplementation_fhnnyx.png)](https://youtu.be/yPecyFKQLuA)
+
+### Disabling Modules
+* Manually Imported Precompiled: Remove the target module files from the build target. This will prevent the modules from being compiled into the release product (most code optimal)
+* Manually Imported Disabled: Alternatively, each module has a ModuleConfig static function where the 'enabled' property can be set to false. Note this will still add the module class files to the end product. Using this method is no longer recommended and may be deprecated in a future release.
+* See the TealiumConfig function options to explicitly white or black list modules for enablement. This option is the most flexible but does not stop unused modules code from being compiled with final product.
```
Comment (tamayok) [inline README.md]: Code sample for this - How-To

File: README.md [hunk 3]
```diff
@@ -7,28 +7,219 @@
...
+------------- | ------------- | ------------------
+config | TealiumConfig object to use. | - 
+
+```swift
+// Simple Sample
+    let config = // See TealiumConfig above
+    let tealium = Tealium(config: config)
+```
+
+```swift
```
Comment (tamayok) [inline README.md]: Add Header + Missing functions?

File: README.md [hunk 4]
```diff
@@ -62,25 +253,57 @@ Logger | Debug logging | 100 | Yes | iOS, macOs, tvOS, watchOs | -
 [Lifecycle](https://community.tealiumiq.com/t5/Mobile-Libraries/Tealium-Swift-Module-Lifecycle/ta-p/16916) | Tracks launches, wakes, sleeps, and crash instances. Auto or manually. | 175 | Yes | iOS, macOS, tvOS, watchOS | -
 Async | Moves all library processing to a background thread | 200 | Yes | iOS, macOS, tvOS, watchOS | -
 [Autotracking](https://community.tealiumiq.com/t5/Mobile-Libraries/Tealium-Swift-Module-Autotracking/ta-p/16856) | Prepares & sends dispatches for most UI, including viewDidAppear, events| 300 | Yes | iOS, tvOS | -
```
Comment (tamayok) [inline README.md]: Periods!

File: README.md (hunk 3 above)
Comment (jalakoo) [inline README.md]: Done!

**Pull request merged**